linera-chain/src/test.rs

```diff
@@ -17,29 +17,31 @@ use linera_execution::{
 
 use crate::{
     block::ConfirmedBlock,
-    data_types::{Block, BlockProposal, IncomingBundle, PostedMessage, SignatureAggregator, Vote},
+    data_types::{
+        BlockProposal, IncomingBundle, PostedMessage, Proposal, SignatureAggregator, Vote,
+    },
     types::{CertificateValue, GenericCertificate},
 };
 
 /// Creates a new child of the given block, with the same timestamp.
-pub fn make_child_block(parent: &Hashed<ConfirmedBlock>) -> Block {
+pub fn make_child_block(parent: &Hashed<ConfirmedBlock>) -> Proposal {
     let parent_value = parent.inner();
-    let parent_block = &parent_value.executed_block().block;
-    Block {
-        epoch: parent_block.epoch,
-        chain_id: parent_block.chain_id,
+    let parent_header = &parent_value.block().header;
+    Proposal {
+        epoch: parent_header.epoch,
+        chain_id: parent_header.chain_id,
         incoming_bundles: vec![],
         operations: vec![],
         previous_block_hash: Some(parent.hash()),
-        height: parent_block.height.try_add_one().unwrap(),
-        authenticated_signer: parent_block.authenticated_signer,
-        timestamp: parent_block.timestamp,
+        height: parent_header.height.try_add_one().unwrap(),
+        authenticated_signer: parent_header.authenticated_signer,
```

Comment:
(Weird! What if this is proposed by someone else? Not new in this PR, of course.)